include/cloysterhpc/presenter/PresenterRepository.h

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2024 Vinícius Ferrão <vinicius@ferrao.net.br>
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#pragma once
```

Comment:
We had this debate, some compilers don't like the `#pragma once`. There's also a Cpp Guideline to use the classic `#ifndef` statement. You can get it from other `.h` files to see the template we use.

Reply:
> We had this debate

Where? (and I am asking without any sarcasm or ill intention)

> some compilers don't like the #pragma once

Every modern and usable compiler supports it. It's basically an unofficial standard at this point.

The only argument against it is handling the `pragma once` across network shares (NFS, SMB or something), but I think we are not sufficiently insane to do this.

> There's also a Cpp Guideline to use the classic #ifndef statement.

Where? (again, just curious about where it is)